iOSSnapshotTestCase.podspec

```diff
@@ -22,7 +22,7 @@ Pod::Spec.new do |s|
   s.pod_target_xcconfig = { 'ENABLE_BITCODE' => 'NO' }
   s.user_target_xcconfig = { 'FRAMEWORK_SEARCH_PATHS' => '$(PLATFORM_DIR)/Developer/Library/Frameworks' }
   s.default_subspecs = 'SwiftSupport'
-  s.swift_version = '4.0'
+  s.swift_version = '4.2'
```

Comment:
I don't see any changes in SwiftSupport.swift, so what benefit does this provide?

Reply:
There are 2 reasons I think:
- Xcode generates warning for targets that don't use latest Swift version
- If we are about to change/add anything in SwiftSupport we want it to be 4.2 compatible